### ISSUE TITLE
Cannot read property 'constructor' of undefined

### ISSUE THREAD

**Issue by lmk123** [TypeStrong/ts-loader#664]
After I upgrade to 3.0.4 it show me the errors:

```
ERROR in ./src/index.ts
Module build failed: TypeError: Cannot read property 'constructor' of undefined
    at Object.getTypeScriptInstance (./node_modules/ts-loader/dist/instances.js:26:37)
    at Object.loader (./node_modules/ts-loader/dist/index.js:24:39)
```

Fallback to 3.0.3 works great.

**Comment by johnnyreilly** [comment]
Please could you provide a minimal repro repo to illustrate this?

**Comment by lmk123** [comment]
After I pinning `chalk` to `v2.2.0` it works!

I'm using node 8

```
$ node -v
8.4.0
```

**Comment by johnnyreilly** [comment]
Hmmm sounds like chalk 2.2 is breaking changes.

**Comment by johnnyreilly** [comment]
Made ts-loader require 2.2 as a minimum and released as [3.0.5](https://github.com/TypeStrong/ts-loader/releases/tag/v3.0.5) 

Thanks for reporting!
